### ISSUE TITLE
Allow to ignore several ports

### ISSUE THREAD

**Issue by frsantos** [Barzahlen/waitforservices#1]
The _-ignoreport_ option permits only one port to be ignored. It could be useful to ignore a list of ports.

**Comment by mfrister** [comment]
Yep, that would be nice. We don't need it right now, so feel free to create a pull request.

**Comment by martinseener** [comment]
@frsantos any updates on this one? We will close this issue if there is no further interest in it.
